app/assets/javascripts/dcv/dcv.date-range.js

```diff
@@ -31,11 +31,12 @@ DCV.DateRangeGraphSelector.lastClickEvent = null;
 DCV.DateRangeGraphSelector.dateCache = null;
 
 DCV.DateRangeGraphSelector.init = function() {
-  if ($('#date-range-widget').length > 0 && DCV.dateWidgetData != null) {
+  var widget = $('#date-range-widget');
```

Comment:
Since this is the same type as `var canvasJQueryElement` below, maybe good to name both in a consistent way (either by changing widget name or canvas name or both).  This is personal preference, but I've gotten into the habit of prefixing jquery element variables with a dollar sign, as in: `$widget` or `$canvas`.

Reply:
Sold (and done).